aries-site/src/examples/templates/popover/PopoverSimpleExample.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useRef } from 'react';
+
+import { Popover } from 'aries-core';
+import { Button, Box, Paragraph, Text } from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+
+export const PopoverSimpleExample = () => {
+  const [showPopover, setShowPopover] = useState(false);
+  const targetRef = useRef();
+
+  const handleButtonClick = () => {
+    setShowPopover(prev => !prev);
+  };
+
+  const handleClosePopover = () => {
+    setShowPopover(false);
+  };
+
+  return (
+    <Box direction="row" justify="center" align="center">
+      <Text>Click for Popover.</Text>
+      <Button
+        align="center"
+        justify="start"
+        icon={<CircleInformation />}
```

Comment:
Can we check screen reader experience? I think the a11yTitle covers us and the announcement of the "Circle information" is unnecessary.

```suggestion
        icon={<CircleInformation aria-hidden="true" />}
```

Reply:
you are correct here double checked on voice over we hear the `informational help` from the button `a11tyTitle`

Reply:
but I can add the `aria-hidden` either way it is covered by the `a11ytitle` in the button